api-tests/api.test.js

```diff
@@ -21,6 +21,14 @@ expect.extend({
       : () => `expected ${received} to be a valid date string`
     return { pass, message }
   },
+  similarDate(recieved, expected) {
+    const parsedRecievedDate = new Date(recieved)
+    const parsedExpectedDate = new Date(expected)
+    return {
+      pass: Number(parsedRecievedDate) == Number(parsedExpectedDate),
```

Comment:
Why `==`?

Reply:
Because I'm used to languages that aren't shitty. Will fix.